profiles/ug/modules/ug/ug_system/ug_system.module

```diff
@@ -15,23 +17,119 @@ function ug_system_menu() {
     'weight' => -10,
   );
 
+  $items['admin/config/system/add-site'] = array(
+    'title' => 'Add site',
+    'description' => 'Add site to lumturio and nagios.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ug_system_add_site_form'),
+    'access arguments' => array('access content'),
+    'weight' => -9,
+  );
   return $items;
 }
 
 /**
- * Form builder; Configure the site's production status.
- *
- * @ingroup forms
- * @see system_settings_form()
+* Form builder
+**/
+function ug_system_add_site_form(){
+  $form['ug_system_add_nagios'] = array(
+    '#type' => 'checkbox',
+    '#name' => 'nagios',
+    '#title' => t('Add site to nagios'),
+    '#description' => t('Send a ticket to the helpdesk for site to be added to nagios'),
+    '#default_value' => variable_get('ug_system_add_nagios', 0),
+  );
+
+  $form['ug_system_add_lumturio'] = array(
+    '#type' => 'checkbox',
+    '#name' => 'lumturio',
+    '#title' => t('Add site to lumturio'),
+    '#description' => t('Add site to lumturio'),
+    '#default_value' => variable_get('ug_system_add_lumturio', 0),
+  );
+  $form['submit'] = array(
+      '#value' => t('Save'),
+      '#type' => 'submit',
+      '#submit' => array('ug_system_form_submit_button'),
+    );
+  return $form;
+}
+
+/**
+* Callback function.
+**/
+function ug_system_form_submit_button($form, &$form_state){
+  if(!empty($_POST["nagios"])) { //Checks if checkbox is empty
+    variable_set('ug_system_add_nagios', 1);
+  }
+  if(!empty($_POST["lumturio"])) { //Checks if checkbox is empty
+    variable_set('ug_system_add_lumturio', 1);
+  }
+  $nagios_checkbox = variable_get('ug_system_add_nagios');
+  if($nagios_checkbox == '1' ) {
+    $to = 'oukanah@icloud.com';
+    $message = drupal_mail('ug_system','ug_system_email',$to, language_default(), 'no-reply@example.com', TRUE); //Sends email
+  }
+
+  if (!empty($message['result'])) {
+    drupal_set_message("Mail sent!");
+  }
+}
+
+/**
+* Implements hook_mail().
+**/
+function ug_system_mail($key, &$message, $params ){
+  switch ($key) {
+    case 'ug_system_email':
+    $message['subject'] = t('add site');
+    $message['body'][] = t('This is an email');
+    $message['body'][] = t('This is an email');
+    break;
+  }
+}
+
+/**
+ * Implements hook_validate().
  */
+function ug_system_add_site_form_validate($form, &$form_state) {
+  if (empty($_POST["nagios"])) {
+    form_set_error('status', t('Please check add site to nagios'));
+  }
+  elseif (empty($_POST["lumturio"])) {
+    form_set_error('status', t('Please check add site to lumturio'));
+  }
+}
+
+/**
+* Implements hook_form_FORM_ID_alter().
+**/
+function ug_system_form_ug_system_add_site_form_alter(&$form, &$form_state, $form_id) {
+  $nagios_checkbox = variable_get('ug_system_add_nagios');
+  $lumturio_checkbox = variable_get('ug_system_add_lumturio');
+  if(($nagios_checkbox and $lumturio_checkbox  == '1')){
```

Comment:
Use && instead of "and".

Reply:
Is the logic correct here? Do we want to disable both checkboxes only if both are checked, or do we want to disable the Lumturio checkbox if it is checked, and the Nagios checkbox if it is checked? Do you need separate conditional statements?